recipes/samurai/all/conanfile.py

```diff
@@ -0,0 +1,99 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy
+from conan.tools.layout import basic_layout
+from conan.tools.scm import Version
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class PackageConan(ConanFile):
+    name = "samurai"
+    description = "Intervals coupled with algebra of set to handle adaptive mesh refinement and operators on it"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/hpc-maths/samurai"
+    topics = ("scientific computing", "adaptive mesh refinement", "header-only")
+    package_type = "header-library"
+    # Keep these or explain why it's not required for this particular case
+    settings = "os", "arch", "compiler", "build_type"
+    # Do not copy sources to build folder for header only projects, unless you need to apply patches
+    no_copy_source = True
+
+    @property
+    def _min_cppstd(self):
+        return "17"
+
+    @property
+    def _compilers_minimum_version(self):
+        return {
+            "Visual Studio": "16.9",
```

Comment:
Where does this number come from? Afaik c++std 17 was not supported in visual studio until 19.0

Reply:
It is ok, /c++17 exists from earlier